interfaces/udev/udev.go

```diff
@@ -0,0 +1,37 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package udev
+
+import (
+	"os/exec"
+)
+
+// ReloadRules runs two commands that reload udev rule database.
+//
+// The commands are: udevadm control --reload-rules
+//                   udevadm trigger
+func ReloadRules() error {
+	err := exec.Command("udevadm", "control", "--reload-rules").Run()
+	if err != nil {
+		return err
+	}
+	err = exec.Command("udevadm", "trigger").Run()
+	return err
```

Comment:
I'm not a udev expert. In general this looks ok but I did notice in the udevadm man page the following:
       -R, --reload
           Signal systemd-udevd to reload the rules files and other databases
           like the kernel module index. Reloading rules and databases does not
           apply any changes to already existing devices; the new configuration
           will only be applied to new events.

I mention this because of the 'existing devices' part. I'm not sure of the implications of that-- I suggest you ask pitti to comment.